### ISSUE TITLE
Missing validation / error handling for object attribute editing

### ISSUE THREAD

**Issue by tschaefer** [zammad/zammad#4490]
### Used Zammad Version

any

### Environment

- Installation method: any
- Operating system: any
- Database + version: any
- Elasticsearch version: any
- Browser + version: any

### Actual behaviour

The user does not get feedback or is prevented from misconfiguration when editing an existing object attribute.

### Expected behaviour

Prevent misconfiguration and show a hint (error message).

### Steps to reproduce the behaviour

* Create a new object attribute from type text with a text length of 120
* Create a new ticket and enter a value (word) into the attribute from the previous step
* Modify the related object attribute and set the text length shorter than the value (word) from the previous step

Clicking the update database button or discarding changes does not work.

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by mgruner** [comment]
@tschaefer @dominikklein @SamolJacek to me it seems this one needs specification / refinement to positively declare what the expected behaviour is, before we can start working on it.
